docs/reference/operator/index.md

```diff
@@ -0,0 +1,3 @@
+## K8sGPT Operator
```

Comment:
I opened up a getting started PR, could either merge or reference that guide at some point

Reply:
I would want this section to detail more on the operator than "here is how you install it"